_pages/clients/python.md

```diff
@@ -64,206 +60,164 @@ client = Algorithmia.client("YOUR_API_KEY", "https://mylocalendpoint")
 {% endhighlight %}
 
-{% endif %}
-
-## Call an Algorithm
+This endpoint will replace a file if it already exists. If you wish to avoid replacing a file, check if the file exists before using this endpoint.
+{: .notice-warning}
 
-Finally we are ready to call an algorithm. In this guide we'll use the natural language processing algorithm called [Summarizer](https://algorithmia.com/algorithms/nlp/Summarizer). This algorithm results in a string that is the summary of the text content you pass in as the algorithm's input.
+You can confirm that the file was created by navigating to Algorithmia's [Hosted Data Source](/data/hosted) and finding your data collection and file.
 
-A single algorithm may have different input and output types, or accept multiple types of input, so consult the algorithm’s description for usage examples specific to that algorithm.
-{: .notice-info}
+You can also upload your data through the UI on Algorithmia's [Hosted Data Source](/data/hosted). For instructions on how to do this go to the [Hosted Data Guide]({{site.baseurl}}/data/hosted).
 
-This example shows the summary of the text file which we downloaded from our data collection and set as the variable called `input` in the previous code sample.
+### Call the Algorithm
 
-Create the algorithm object and pass in the variable `input` into `algo.pipe()`:
+Once the file has been uploaded, you are ready to call the algorithm. Create the algorithm object and pass the URI (which is stored in `img_file` in the code above) for your file as an argument to `algo.pipe()`:
 
 {% highlight python %}
 # Create the algorithm object using the Summarizer algorithm
-algo = client.algo('nlp/Summarizer/0.1.3')
+image_algo = client.algo('*TODO*')
 # Pass in input required by algorithm
 try:
     # Get the summary result of your file's contents
-    print(algo.pipe(input).result)
+    response = algo.pipe(img_file)
 except Exception as error:
     # Algorithm error if, for example, the input is not correctly formatted
     print(error)
 {% endhighlight %}
 
-This guide used the the first chapter of [Jack London's Burning Daylight](https://en.wikisource.org/wiki/Burning_Daylight) and the Summarizer algorithm outputs:
-
-"It was a quiet night in the Shovel. The miners were in from Moseyed Creek and the other diggings to the west, the summer washing had been good, and the men's pouches were heavy with dust and nuggets. MacDonald grinned and nodded, and opened his mouth to speak, when the front door swung wide and a man appeared in the light."
-
-If you are interested in learning more about working with unstructured text data check out our guide [Introduction to Natural Language Processing](https://algorithmia.com/blog/introduction-natural-language-processing-nlp).
+Once the algorithm has completed, `response.result` will contain the URI for the resulting file, which you can then download (or provide as input to another algorithm in a pipeline).
 
-### Limits
```

Comment:
This shouldn't be removed - didn't see it in the new version unless i skipped it